src/SignalR/clients/java/signalr/src/main/java/com/microsoft/signalr/HubConnection.java

```diff
@@ -136,7 +136,7 @@ Transport getTransport() {
         }
 
         this.baseUrl = url;
-        this.protocol = new JsonHubProtocol();
+        this.protocol = new MessagePackHubProtocol();
```

Comment:
Change this

Reply:
@BrennanConroy @halter73 should this just be a param to the HubConnection constructor?

Reply:
We'll want a method on the builder similar to https://github.com/dotnet/aspnetcore/blob/50fb513fdae68d68bf81a12b049e6ae0345c9a2c/src/SignalR/clients/ts/signalr/src/HubConnectionBuilder.ts#L147
and yes, the protocol will be passed to the constructor.

Reply:
https://github.com/dotnet/aspnetcore/pull/23532/commits/a05868fce24989c20bd96f4001f4c3eaa2753644

Reply:
Need to default it to JsonHubProtocol :D